app/services/identity_reconciler.go

```diff
@@ -62,6 +63,11 @@ func IdentityReconciler(accountStore data.AccountStore, cfg *app.Config, provide
 	if err != nil {
 		return nil, errors.Wrap(err, "AccountCreator")
 	}
-	accountStore.AddOauthAccount(newAccount.ID, providerName, providerUser.ID, providerToken.AccessToken)
+	err = accountStore.AddOauthAccount(newAccount.ID, providerName, providerUser.ID, providerToken.AccessToken)
+	if err != nil {
```

Comment:
This is probably the most interesting thing turned up here.  I could see an argument to continue ignoring as well.

Reply:
Passing it back to the caller makes sense to me. I don't see how it will happen but if it does this will be necessary to discover and fix.